integration/runner/runner.go

```diff
@@ -204,7 +204,7 @@ func Run() error {
 
 	// Build cAdvisor.
 	glog.Infof("Building cAdvisor...")
-	err := RunCommand("godep", "go", "build", "github.com/google/cadvisor")
+	err := RunCommand("godep", "go", "build", "--ldflags='-extldflags \"-static\"'", "github.com/google/cadvisor")
```

Comment:
Did you try running `make build`?

On Tue, May 3, 2016 at 12:23 PM, Lucas Käldström <notifications@github.com>
wrote:

> In integration/runner/runner.go
> <https://github.com/google/cadvisor/pull/1237#discussion_r61940055>:
>
> > @@ -204,7 +204,7 @@ func Run() error {
> >
> >  	// Build cAdvisor.
> >  	glog.Infof("Building cAdvisor...")
> > -	err := RunCommand("godep", "go", "build", "github.com/google/cadvisor")
> > +	err := RunCommand("godep", "go", "build", "--ldflags='-extldflags \"-static\"'", "github.com/google/cadvisor")
>
> @vishh <https://github.com/vishh> Do you have any clue what to write here
> to make it work?
> I tried a lot of different combinations, but the linker just didn't parse
> it.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/google/cadvisor/pull/1237/files/c7e2e113a9084142964556bd76b43ffb64b9e0e9#r61940055>
>